MultiplexReservationSystem/src/main/java/p/lodz/pl/multiplexreservationsystem/model/TicketType.java

```diff
@@ -0,0 +1,14 @@
+package p.lodz.pl.multiplexreservationsystem.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum TicketType {
+  ADULT(25),
```

Comment:
Czyli teraz zmiana cen biletu (inflacja, więc często!) powoduje konieczność zrekompilowania aplikacji?

Reply:
Zmiana w deklaracjach/sygnaturach metod i zmiennych składowych klas rzeczywiście powoduje potrzebe zrekompilowania aplikacji. Zmiana w ciele metody wymaga rekompilacji samej klasy.
Czy ten problem możnaby było rozwiązać poprzez dodanie konstruktora do TicketType z następującym ciałem? :
this.price = switch(type) {
    case ADULT -> 25;
    case STUDENT -> 18;
    case CHILD -> 12.5; 
};